packages/site/e2e/layout.detailHighlight.test.ts

```diff
@@ -1,9 +1,10 @@
 import { test, expect } from '@playwright/test';
 
+import { waitForFonts } from './utils';
+
 test('take a screenshot', async ({ page }) => {
   await page.goto('/mosaic/test/layouts/detail-highlight');
-  await page.evaluate(() => document.fonts.ready);
-  await page.waitForFunction(() => document.fonts.check('18px Open Sans'));
+  await page.waitForFunction(waitForFonts);
```

Comment:
would globalSetup work for this ?

Reply:
might do, but can't get the tests to pass at the moment.  Fonts aren't loaded correctly in the test